src/wp-includes/meta.php

```diff
@@ -1202,6 +1253,7 @@ function register_meta( $object_type, $meta_key, $args, $deprecated = null ) {
 	 * @param string $meta_key    Meta key.
 	 */
 	$args = apply_filters( 'register_meta_args', $args, $defaults, $object_type, $meta_key );
+	unset( $defaults['default'] );
```

Comment:
What about removing the `default` entry from `$defaults` entirely? It doesn't seem meaningful to include it in the array if it doesn't indicate anything.

Reply:
It is need because this filter - https://github.com/WordPress/wordpress-develop/blob/8b9823f53621a63d0a47626920f6930f35db160c/src/wp-includes/meta.php#L1410-L1412

Reply:
Aha! Thanks for clarifying.